joomla/joomla-cms
# [5.1] Load plugin group when executing batch tasks

### Summary of Changes
@grantg182 found an issue in J3 that batch operations aren't loading the content plugin group. He provided a quick fix in #38378, but since we don't do bugfixes in J3 anymore, this is unfortunately not acceptable anymore. However, this is still valid in J4 and thus this PR introduces loading these plugins in batch properly.

### Testing Instructions
Add a `die();` in the beginning of a content plugin. Batch move a few articles. See that it goes through. Apply patch. Move the articles again. See that you get a white screen because the content plugins are loaded.

### Link to documentations
Please select:
